src/uct/ib/base/ib_device.c

```diff
@@ -27,6 +27,16 @@ typedef struct {
 } uct_ib_device_gid_info_t;
 
 
+/* this table is according to Table "Encoding for RNR
```

Comment:
This table is according to "Encoding for RNR NAK Timer Field" in IBTA specification

Reply:
done